ozaria/site/components/teacher-dashboard/BaseTeacherDashboard/index.vue

```diff
@@ -53,7 +57,8 @@ export default {
       // // We may want to pull this out. For locality with dashboard this reduces abstraction.
       runningTour: null,
       createdFirstClass: false,
-      trialRequestLoading: true
+      trialRequestLoading: true,
+      newClassroom: new Classroom({ ownerID: me.id })
```

Comment:
why vue and backbone interpolation?

Reply:
same as above. the modal that will process it, requires this type.